src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
Since the formatter is a constant and will not change, it should be declared as a static final field and follow the naming conventions for constants (all uppercase letters with underscores). Consider renaming it to 'DATE_FORMATTER'.